### PR TITLE
Completed html-tables exercise.

### PR DESCRIPTION
**Pull request by Yang-Darren**
image

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Tables</title>
+</head>
+
+<body>
+  <h1>Student Grade Table</h1>
+  <table>
```
Comment (BrettAlbright) [inline html-tables/index.html:10]: make sure to include the `thead` and `tbody` elements into your table.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

File: html-tables/index.html (hunk 1 above)
Comment (Yang-Darren) [reply html-tables/index.html:10]: image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**